### ISSUE TITLE
[MCHANGES-72] Build Failure using IBM JDK 1.4.2 SR7

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-changes-plugin#392]
**[Patrick Schneider](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=pschneider)** opened **[MCHANGES-72](https://issues.apache.org/jira/browse/MCHANGES-72?redirect=false)** and commented

Attached is the -e output from running 'mvn site:site'.

---

**Affects:** 2.0-beta-2

**Attachments:**
- [changes-ibmjdk.txt](https://issues.apache.org/jira/secure/attachment/12714503/changes-ibmjdk.txt) (_5.86 kB_)
- [MCHANGES-72.patch](https://issues.apache.org/jira/secure/attachment/12714636/MCHANGES-72.patch) (_1.68 kB_)

**Issue Links:**
- [MCHANGES-182](https://issues.apache.org/jira/browse/MCHANGES-182) Plugin seems to be tied to Sun JDK
 (_**"is duplicated by"**_)

2 votes, 2 watchers

**Comment by jira-importer** [comment]
**[Stephane Nicoll](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=snicoll)** commented

decreased priority until we get feedback.

**Comment by jira-importer** [comment]
**[Dennis Lundberg](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dennisl@apache.org)** commented

Here's a proposed patch that at least should provide a more user friendly error message. I don't have the necessary tools to try it out myself though.

**Comment by jira-importer** [comment]
**[Chris Graham](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=chrisgwarp)** commented

To answer Dennis's question on 7/7/7:

This is under AIX 5.3:

```
>>java -version
java version "1.5.0"
Java(TM) 2 Runtime Environment, Standard Edition (build pap64devifx-20071025 (SR6b))
IBM J9 VM (build 2.3, J2RE 1.5.0 IBM J9 2.3 AIX ppc64-64 j9vmap6423-20071007 (JIT enabled)
J9VM - 20071004_14218_BHdSMr
JIT  - 20070820_1846ifx1_r8
GC   - 200708_10)
JCL  - 20071025
>>java -fullversion
full version: J2RE 1.5.0 IBM AIX build jclap64devifx-20071025 java - J2RE 1.5.0 IBM AIX build jclap64devifx-20071025 java
>>oslevel -r
5300-06
```

Yes, it occurs when issuing 'mvn changes:announcement-mail'

In a parent pom:

```
    <properties>
        <maven-changes-plugin.version>2.1</maven-changes-plugin.version>
    </properties>

    <build>
        <pluginManagement>
            <plugins>
                <plugin>
                    <artifactId>maven-changes-plugin</artifactId>
                    <version>${maven-changes-plugin.version}</version>
                    <inherited>true</inherited>
                    <configuration>
                        <smtpHost>sctmailhost</smtpHost>
                        <mailSender>
                            <name>Release Notification</name>
                            <email>me@here.com</email>
                        </mailSender>
                        <toAddresses>
                            <toAddress>me@here.com</toAddress>
                        </toAddresses>
                    </configuration>
                </plugin>
            </plugins>
        </pluginManagement>
    </build>
```

In the local pom:

```
<build>
    <plugins>
        <plugin>
            <groupId>org.apache.maven.plugins</groupId>
            <artifactId>maven-changes-plugin</artifactId>
            <configuration>
                <mailSender>
                    <name>Release Notification</name>
                    <email>me@here.com</email>
                </mailSender>
            </configuration>
        </plugin>
    </plugins>
</build>
```

So, SSL is not explicitly set.

If you need more info, then please ask.

**Comment by jira-importer** [comment]
**[Dennis Lundberg](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dennisl@apache.org)** commented

I have applied my patch with improved error message in [r1050533](http://svn.apache.org/viewvc?view=revision&revision=1050533), and it will be included in the 2.4 release.

**Comment by jira-importer** [comment]
**[Dennis Lundberg](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=dennisl@apache.org)** commented

In [r1050951](http://svn.apache.org/viewvc?view=revision&revision=1050951) I upgraded the plexus-mail-sender components to version 1.0-alpha-7, hoping that it will help this issue. A new 2.4-SNAPSHOT has been deployed. Please test it if you the possibility.

**Comment by jira-importer** [comment]
**[Benson Margulies](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=bmargulies)** commented

This was fixed back in 2.4 and never marked closed.
